camus-kafka-coders/src/main/java/com/linkedin/camus/etl/kafka/coders/JsonStringMessageDecoder.java

```diff
@@ -120,15 +120,18 @@ else if (timestampFormat.equals("ISO-8601")) {
         }
       }
     }
+    else {
+      log.warn("Message does not have the field " + timestampField); // TODO
```

Comment:
I like it, another datadog gauge?

Reply:
We could, but then I'd need to store another flag in the wrapper. For now I just wanted to validate that when the warning was printed it was because of this.

Reply:
gotcha